data/multiplayer/scenarios/2p_Isle_of_Mists.cfg

```diff
@@ -0,0 +1,298 @@
+#textdomain wesnoth-IoM
+
+#define ISLE_OF_MISTS_MSG_STR
+    _ "Your aim is to survive spawning waves of units and defeat the final archenemies. Spawns are randomly generated and will be different each time Isle of Mists is played. The waves appear at somewhat regular intervals. The weather will also change randomly, affecting the layout of the map."
+#enddef
+
+[multiplayer]
+    id=multiplayer_2p_Isle_of_Mists
+    name= _ "2p — Isle of Mists (Survival)"
+    description= _ "Isle of Mists is a survival scenario for solitaire or two-player team-based play against randomly-spawned AI units, catered for experienced players. Victory is achieved by surviving and defeating all enemy waves. During the course of play, the terrain will change based on random weather effects. Default map settings are required for the scenario to work properly. By Lord-Knightmare."
```

Comment:
Remove all the dashes from the first sentence.  Also, "solitaire" is a strange word to use here.  I'd say:
> Isle of Mists is a survival scenario which can be played alone or with another player against randomly spawned AI units ...

Reply:
Done